### PR TITLE
Micronaut: Add missing path variable parameter fix.

### PR DESCRIPTION
**Pull request by dbalek**
Micronaut reports error if a path variable is declared the URL path and the corresponding parameter is missing in a method signature. The quick-fix adds the missing parameter.
```
  @Get("/path/{variable}/")
  public String handle(String name_not_equal_to_variable) {
    ...
  }
```

### CODE REVIEW

**Comment by mbien** [comment]
do the micronaut tests require a JDK 17 setup? They are still green but there are exceptions in the log:

```
    [junit] Caused by: java.lang.UnsupportedClassVersionError: io/micronaut/maven/testresources/TestResourcesLifecycleExtension has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0
    [junit] 	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
    [junit] 	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1022)
```

**Review by MartinBalin — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -2198,6 +2195,15 @@ jobs:
       - name: j2ee.dd.webservice
         run: ant $OPTS -f enterprise/j2ee.dd.webservice test
 
+      - name: Set up JDK 17 for incompatible tests
```
Comment (lahodaj) [inline .github/workflows/main.yml:2198]:
```suggestion
      - name: Set up JDK 17 for tests that are not compatible with JDK 11
```

**Review by lahodaj — APPROVED** [review]
Looks reasonable to me. One suggestion inline.

File: .github/workflows/main.yml (hunk 1 above)
Comment (dbalek) [reply .github/workflows/main.yml]: Done

**Pull request merged**